src/apis/getItems.ts

```diff
@@ -0,0 +1,13 @@
+import { API_KEY, QUERY_TYPE } from '@/constants/constantValues';
+
+export const getItems = (queryType: string, contentType: string, search: string) => {
+  const QUERY_TYPE_INFO = {
+    [QUERY_TYPE.TOP_RATED]: `https://api.themoviedb.org/3/${contentType}/top_rated?api_key=${API_KEY}&language=en-US`,
+    [QUERY_TYPE.SEARCH]: `https://api.themoviedb.org/3/search/${contentType}?api_key=${API_KEY}&language=en-US&query=${search}`,
+  };
+
+  return fetch(QUERY_TYPE_INFO[queryType]).then((response) => {
```

Comment:
If you compare `getItem` and `getItems`, you have repetition in the code. What about writing these two functions as **async** ones?

Reply:
Can I leave them as separate functions because they have different logic for creating URLs, but make helper functions for getting data and handling responses?

Reply:
Create get function and then reuse it in both of them.